name: "Testing"

on:
  pull_request:
    types: [ auto_merge_enabled ]

jobs:
  test:
    name: Test
    runs-on: ubuntu-latest
    steps:
      - name: check
        run: echo "${{ context.payload.pull_request.auto_merge }}"
    
      - uses: actions/checkout@v3
    
      - name: Test
        uses: actions/github-script@v6
        with:
          script: |
            const { test, test2 } = require('./.github/scripts/index.js');
            await test({github, context});
            await test2({github, context});

            const log_test1 = "logged_value1";
            const log_test2 = "logged_value2";
            core.info('Output to the actions build log', log_test1);
            core.info('Output to the actions build log' + log_test1 + 'more text: ' + log_test2);
            core.notice('Something happened that you might want to know about.');
            core.warning('Something went wrong, but it\'s not bad enough to fail the build.');
            core.error('This is a bad error, action may still succeed though.');
            

# const { test, test2 } = require('.github/scripts/index.js');
#await test({github, context});
#            await test2({github, context})
#const utils = require('./.github/scripts/utils.js')
#const test = require('./.github/scripts/test.js')
